src/utils/convert.js

```diff
@@ -53,6 +53,28 @@ exports.default = {
         }
         return bytes;
     },
+    amountToByteArray: function (input) {
+        if (typeof input !== 'number') {
+            throw new Error('Numeric input is expected');
+        }
+        var bytes = new Array(7);
+        for (var k = 7; k >= 0; k--) {
+            bytes[k] = input & (16);
+            input = input / 16;
+        }
+        return bytes;
+    },
```

Comment:
where to use the `amountToByteArray`?

btw, to byte (2^8 = 256) array, `input & (16)` and `input / 16` are incorrect